src/devices/Bmxx80/Bme680.cs

```diff
@@ -583,5 +583,65 @@ private byte CalculateHeaterDuration(ushort duration)
 
             return durationValue;
         }
+
+        private bool TryReadHumidityInternal(out double humidity, bool skipTempFineRead = false)
+        {
+            if (HumiditySampling == Sampling.Skipped)
+            {
+                humidity = double.NaN;
+                return false;
+            }
+
+            // Read humidity data.
+            var hum = Read16BitsFromRegister((byte)Bme680Register.HUMIDITYDATA, Endianness.BigEndian);
+
+            if (!skipTempFineRead)
+            {
+                TryReadTemperatureInternal(out _);
```

Comment:
should you return false when this returns false?

Reply:
No, this method is called to update a value wich is used for internal calculation purposes. If it returns false the value won't be updated but that's expected and is not necessarily bad. Per default sensor configuration this will always return true, only if the user chooses to skip temperature measurements will this value not get updated and this is often a conscious decission to get results faster or if the temperature doesn't change a lot.